runtime/src/lib.rs

```diff
@@ -13,6 +13,16 @@
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
+#[cfg(feature = "std")]
+/// Wasm binary unwrapped. If built with `BUILD_DUMMY_WASM_BINARY`, the function panics.
```

Comment:
Do we need to remove this line from our README now ?
https://github.com/Joystream/joystream/blob/master/README.md#rls-extension-in-vscode-or-atom-editors

Reply:
`cargo clippy` worked in my CLion without the `BUILD_DUMMY_WASM_BINARY`. I will try again on building the v2.0.1

Reply:
yeah we still need it in VSCode ..